calendar.ts

```diff
@@ -0,0 +1,209 @@
+enum MonthValues {
```

Comment:
hmmm? what's with the weird numbers

Reply:
[Key-Value](http://mathforum.org/dr.math/faq/faq.calendar.html) method for determining the day of the week